src/workerd/jsg/resource.h

```diff
@@ -1531,7 +1531,10 @@ class ResourceWrapper {
 
     instance->SetInternalFieldCount(Wrappable::INTERNAL_FIELD_COUNT);
 
-    constructor->SetClassName(v8StrIntern(isolate, typeName(typeid(T))));
+    auto classname = v8StrIntern(isolate, typeName(typeid(T)));
```

Comment:
Q: Does this add SymbolToStringTag to all resources?

Reply:
It adds it to everything that extends from `jsg::Object` yes